### PR TITLE
[risk=low][no ticket] Updates to ManageLeonardoRuntimes

### PR DESCRIPTION
**Pull request by jmthibault79**
I had an occasion to use these tools while helping a user, so I make some small fixes.

Tested by creating a runtime and running these commands locally:
```
./project.rb list-runtimes --project=all-of-us-rw-preprod --google-project terra-vpc-sc-7d68ac3f
./project.rb describe-runtime --project=all-of-us-rw-preprod --id terra-vpc-sc-7d68ac3f/all-of-us-45
./project.rb delete-runtimes --project=all-of-us-rw-preprod --ids terra-vpc-sc-7d68ac3f/all-of-us-45
```

---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/build.gradle [hunk 1]
```diff
@@ -729,7 +729,9 @@ task loadConfig(type: JavaExec) {
 }
 
 // Called by devs from the command-line:
-// - devstart.rb > list_runtimes
+// ./project.rb list-runtimes
```
Comment (jmthibault79) [inline api/build.gradle:732]: This one gradle tool powers 3 project.rb commandlines

File: api/libproject/devstart.rb [hunk 2]
```diff
@@ -1914,7 +1910,7 @@ def describe_runtime(cmd_name, *args)
     common = Common.new
     common.run_inline %W{
        ./gradlew manageLeonardoRuntimes
-      -PappArgs=['describe','#{api_url}','#{gcc.project}','#{ctx.service_account}','#{op.opts.runtime_id}']}
```
Comment (jmthibault79) [inline api/libproject/devstart.rb:1917]: not just unused, but wrong: this was attempting to pass the `environment` as the workspace google proejct

File: api/libproject/devstart.rb [hunk 3]
```diff
@@ -1929,19 +1925,19 @@ def list_runtimes(cmd_name, *args)
   op = WbOptionsParser.new(cmd_name, args)
   gcc = GcloudContextV2.new(op)
   op.add_option(
-      "--runtime-project [project]",
```
Comment (jmthibault79) [inline api/libproject/devstart.rb:1932]: "runtime project" is ambiguous: do you mean the billing project (AKA workspace namespace) or the google project?

I think think is a leftover from the time when these values were equal.  today they are not.

File: api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java [hunk 4]
```diff
@@ -67,7 +67,7 @@ public ManageLeonardoRuntimes(LeonardoMapper leonardoMapper) {
         "https://www.googleapis.com/auth/userinfo.email"
       };
   private static final List<String> DESCRIBE_ARG_NAMES =
-      ImmutableList.of("api_url", "project", "service_account", "runtime_id");
+      ImmutableList.of("api_url", "service_account", "runtime_id");
```
Comment (jmthibault79) [inline api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java:70]: run unused

File: api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java [hunk 5]
```diff
@@ -249,7 +249,7 @@ public CommandLineRunner run() {
                     "Expected %d args %s. Got: %s",
                     DESCRIBE_ARG_NAMES.size(), DESCRIBE_ARG_NAMES, Arrays.asList(args)));
           }
-          describeRuntime(args[0], args[2], args[3]);
+          describeRuntime(args[0], args[1], args[2]);
```
Comment (jmthibault79) [inline api/tools/src/main/java/org/pmiops/workbench/tools/ManageLeonardoRuntimes.java:252]: shifted by 1 because project was unused

**Review by dmohs — APPROVED** [review]
Great inline comments.

**Pull request merged**